### PR TITLE
Prachi

### PR DESCRIPTION
**Pull request by patmasid**

### CODE REVIEW

File: README.md
Comment (tommy-waltmann) [inline README.md:1]:
```suggestion
# voting-ml
```

File: README.md
Comment (tommy-waltmann) [inline README.md:6]:
```suggestion
git fetch --recurse-submodules
```

File: output/test/ftsel_questions_list.txt
Comment (tommy-waltmann) [inline output/test/ftsel_questions_list.txt:1]: Out of curiosity, what is this list of columns for?

File: voting_ml/data.py
Comment (tommy-waltmann) [inline voting_ml/data.py:80]:
```suggestion
            return_table = np.empty((self._dataframe.shape[0], len(question_list)), dtype=object)
```

File: voting_ml/data.py
Comment (tommy-waltmann) [inline voting_ml/data.py:101]:
```suggestion
```

File: voting_ml/data.py
Comment (tommy-waltmann) [inline voting_ml/data.py:121]:
```suggestion


```

File: voting_ml/data.py
Comment (tommy-waltmann) [inline voting_ml/data.py:125]:
```suggestion

```

File: voting_ml/data.py
Comment (tommy-waltmann) [inline voting_ml/data.py:24]: If this is variable is not used anywhere else, we should remove it.

File: voting_ml/example_data_access.py
Comment (tommy-waltmann) [inline voting_ml/example_data_access.py:30]: Please add back in the call to `all_data`. I don't want you to remove your call to `all_data_except`, I would rather have both of them in the file.

File: voting_ml/example_data_access.py
Comment (tommy-waltmann) [inline voting_ml/example_data_access.py:34]: Why change this from `True` to `False`?

File: voting_ml/example_data_access.py
Comment (tommy-waltmann) [inline voting_ml/example_data_access.py:3]: Just a general comment about this file: It should serve as a mini unit test of the `PollDataProxy` class. I should be able to run this file, inspect the output, and be confident the class is working properly. That means we shouldn't remove things that are already in this file, rather we should add things that are not here as we add methods/functionality to the class.

File: voting_ml/feature_selection.py
Comment (tommy-waltmann) [inline voting_ml/feature_selection.py:24]: I don't think this needs to be here.

**Review by tommy-waltmann — CHANGES_REQUESTED** [review]
There are two things that I want to address in addition to my comments:

1) You seem to have removed the poll data from FiveThrityEight as a submodule, that MUST be added back before this PR can be merged.

2) Please do not commit anything in a `__pycache__` directory. Right now there are two files which have been committed like this. You can unstage those files, and also add to the `.gitignore` a single line which has `__pycache__/` in it so this won't happen again. If you need help doing this I am happy to provide it.

File: output/test/ftsel_questions_list.txt
Comment (patmasid) [reply output/test/ftsel_questions_list.txt:1]: This is the output of my class feature_selection. These are the features that have been selected in that run.

File: voting_ml/data.py
Comment (patmasid) [reply voting_ml/data.py:101]: Because in the feature selection, we do ordinal encoding, we might not need to convert to integers. But if we want to convert it before, it should be okay. That's why I had removed it for now.

File: voting_ml/example_data_access.py
Comment (patmasid) [reply voting_ml/example_data_access.py:30]: Sorry for this. I was just checking some things I will add everything.

File: voting_ml/example_data_access.py
Comment (patmasid) [reply voting_ml/example_data_access.py:34]: Again, I was checking something. I will add this back.

File: voting_ml/example_data_access.py
Comment (patmasid) [reply voting_ml/example_data_access.py:3]: The file in the branch is not supposed to be a final one. We can merge the two files, which was my point of having two branches.

**Pull request merged**
